src/proof/lean/lean_printer.h

```diff
@@ -14,23 +14,36 @@
 
 #include "cvc4_private.h"
 
-#ifndef CVC4__PROOF__LEAN_PROOF_POST_PROCESSOR_H
-#define CVC4__PROOF__LEAN_PROOF_POST_PROCESSOR_H
+#ifndef CVC4__PROOF__LEAN_PROOF_PRINTER_H
+#define CVC4__PROOF__LEAN_PROOF_PRINTER_H
+
+#include <iostream>
+#include "expr/proof_node.h"
+
+namespace CVC4 {
+namespace proof {
 
 enum LeanRule
 {
+  // in what format should I put these lean rules
```

Comment:
Let's try to follow the enumeration in `expr/proof_rule.h`. So an enum value for each lean of the Lean calculus.

Reply:
Can we make this an enum class?

Reply:
Yes, we should. So it should change to `enum class LeanRule : uint32_t`.